Espanol/SnowWhiteDwarf_es.ipynb

```diff
@@ -503,7 +506,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "logg = pd.read_csv('dat/logg.txt', sep='\\s+', comment='#', names=['Cuerpo', 'Gravedad en superficie [g]'])\n",
+    "logg = pd.read_csv('../dat/logg.txt', sep='\\s+', comment='#', names=['Cuerpo', 'Gravedad en superficie [g]'])\n",
```

Comment:
Sorry, but I'm confused this is what I already see in binder using the master branch. As well as some of the other changes...

Reply:
It seems that these changes are not reflected on binder. In the Figure you can see the folders and files that I see in this folder on binder.

![image](https://user-images.githubusercontent.com/3946766/100259138-df2cb580-2f15-11eb-818d-f65634227774.png)